docs/source/conf.py

```diff
@@ -50,7 +50,7 @@
 master_doc = "index"
 
 # General information about the project.
-project = "mypy"
+project = "basedmypy"
 copyright = "2012-2022 Jukka Lehtosalo and mypy contributors"
```

Comment:
does this need to be edited too? maybe we need a lawyer to review the docs

Reply:
No idea, lets leave this thread open so we don't forget about it.

Reply:
https://github.com/KotlinIsland/basedmypy/issues/478